main.go

```diff
@@ -2804,6 +2810,61 @@ The following options are recognized:
 		return nil
 	})
 
+	r.Dispatch("uuid", &Help{
+		Summary: "Generate a version 4 uuid value and write it to the specified path and/or key. If no key is provide the key `uuid` is used",
+		Usage:   "safe uuid PATH[:KEY]",
+		Type:    NonDestructiveCommand,
+		Description: ``,
+	}, func(command string, args ...string) error {
+		rc.Apply(opt.UseTarget)
+
+		if len(args) == 0 {
+			r.ExitWithUsage("uuid")
+		}
+
+		u := uuid.NewRandom()
+
+		stringuuid := u.String()
+
+		v := connect(true)
+
+		for len(args) > 0 {
```

Comment:
Because the semantics of whether providing multiple path arguments would be unclear as to whether it should generate different UUIDs for each or not, let's just assert that exactly one arg was given and drop this loop.

Reply:
Do you recommend we check to see if only one path was given and if not throw an error?

Reply:
yes. it might be expanded in the future to support multiple paths, but for now, just err if there's not exactly one arg.